org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/wizards/MavenProjectWizardArchetypePage.java

```diff
@@ -438,7 +438,7 @@ public void inputChanged(Viewer viewer, Object oldInput, Object newInput) {
         String description = archetype.getDescription();
 
         String text = description == null ? "" : description; //$NON-NLS-1$
-        text = text.replaceAll("\n", "").replaceAll("\\s{2,}", " "); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$ //$NON-NLS-4$
+        text = text.replace("\n", "").replaceAll("\\s{2,}", " "); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$ //$NON-NLS-4$
```

Comment:
seems wrong

Reply:
my bad, it's fine

Reply:
Exactly the phases I passed myself when first seeing that ...